pkg/template/types.go

```diff
@@ -25,8 +25,11 @@ type Params struct {
 	// the Ingress Controller service. This allows us to migrate providers to
 	// chart-operator independently.
 	DisableIngressControllerService bool
-	Etcd                            Etcd
-	Extension                       Extension
+	// DockerhubToken is an auth token used by kubelet to
+	// authenticate/authorize against https://index.docker.io/v1/.
+	DockerhubToken string
```

Comment:
Don't we need to fail when it's empty (if so that would be a breaking change)?

Reply:
We need to fail 100%.